Dockerfile

```diff
@@ -1,26 +1,21 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including some of the settings below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
+# The playbook to be run is specified via the PLAYBOOK_FILE env var
+# Setting a default of openshift_facts.yml as it's an informative playbook
+# that can help test that everything is set properly (inventory, sshkeys)
+ENV PLAYBOOK_FILE=plabyooks/byo/openshift_facts.yml \
+    ANSIBLE_HOST_KEY_CHECKING=False \
```

Comment:
Can we provide a user to run the system container as and automatically mount ~/.ssh for that user?

Then we could provide a flag to opt in for host key checking.

This would also help for ssh proxy config, environment-specific auth, etc.